common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftAuthorizationResponse.java

```diff
@@ -85,6 +105,15 @@ public void setCorrelationId(final String correlationId) {
 
     public String getSessionState() { return mSessionState;}
 
+    public String getDeviceCode() { return mDeviceCode;}
```

Comment:
Most of these properties are going to be unfamiliar to us, as we're just now adding support for DCF. Despite the other getters not having javadoc in this class, can I request we add it for these new methods? I think it's is particularly important to call out that these getters and their values will return `null` in non-DCF cases, and that these values may be applicable only in DC flows

Reply:
done!